quickstart/targets.go

```diff
@@ -0,0 +1,68 @@
+package quickstart
+
+import (
+	"fmt"
+
+	"github.com/charmbracelet/huh"
+	"github.com/pkg/errors"
+	"github.com/speakeasy-api/openapi-generation/v2/pkg/generate"
+	"github.com/speakeasy-api/sdk-gen-config/workflow"
+)
+
+func targetBaseForm(inputWorkflow *workflow.Workflow) (*State, error) {
+	var targetName, targetType, sourceName, outputLocation string
+	if err := huh.NewForm(
```

Comment:
I think we should probably wrap all forms in our own custom models. It will make things like titles looks a lot better than what I laid out here.